### DISCUSSION TITLE
AnalyticalTable: Aggregate property in the analytical table columns

### DISCUSSION THREAD

**Discussion by pppwr7** [UI5/webcomponents-react#6758] [Q&A] [answered]
Hi Colleagues,
I see that the analytical table columns have property like aggregate, Aggregated and aggregateValue. I dont find any example on how to use this property. Could anybody please help me with an example about this. I have a requirement to show the total of the quantities at the bottom of the table column. This total is brought in via another field and is not supposed to be caluclated out of the currently loaded rows. Please let me know if such feature is available currently.

**Comment by Lukas742** [comment, marked as answer]
Hi @pppwr7 

this behavior is currently not supported by the `AnalyticalTable`, but we'll take it for consideration in the new table PoC we're currently implementing. 
As a workaround, you can preprocess the `data` array and add the sum of each column as last entry.
